test/functional/command/runner_test.py

```diff
@@ -65,11 +65,11 @@ def test_create_docker_runner_cache_shared():
     os=Linux
     [runner]
     type=docker
```

Comment:
seeing how the profile looks like, I think it would be better use the `type` field to concatenate to the other fields of the config. It is strange to see `type=docker` and then all the other fields starting with `docker.`